server/lobby.js

```diff
@@ -227,7 +228,7 @@ class Lobby {
         this.sockets[ioSocket.id] = socket;
 
         if(socket.user) {
-            this.users[socket.user.username] = socket.user;
+            this.users[socket.user.username] = Util.getUserWithDefaultsSet(socket.user);
```

Comment:
They're different user objects, one is the one we trust from the client, one is the one we get from the database